client.lua

```diff
@@ -1071,19 +1073,33 @@ local function SpawnPed(data)
 		if nextnumber <= 0 then nextnumber = 1 end
 
 		Config.Peds[nextnumber] = data
+		createdPeds[#createdPeds + 1] = data.currentpednumber
```

Comment:
Could instead return early with the singular ped ID since `data` wouldn't be a table of peds in this instance.